networkx/drawing/nx_pylab.py

```diff
@@ -192,8 +192,8 @@ def draw_networkx(G, pos=None, arrows=True, with_labels=True, **kwds):
     linewidths : [None | scalar | sequence]
        Line width of symbol border (default =1.0)
 
-    width : float, optional (default=1.0)
-       Line width of edges
+    width : float, or array of floats
+       Line width of edges (default=1.0)
```

Comment:
```suggestion
    width : float or array of floats (default=1.0)
       Line width of edges
```
I suggest leaving the default on the parameter-type line to stay consistent with all of the other entries.

Reply:
Yes, the defaults should be on the first line -- but I see that the `linewidth` parameter just above this one has it on the second line and the `width` parameter to `draw_networkx_edges()` is exactly the same as this one is now -- default on the second line. Looks like we should clean up the docstrings in this module.